test/libsolidity/syntaxTests/storageLayoutSpecifier/bitwise_negation_after_cast.sol

```diff
@@ -1,3 +1,3 @@
 contract C layout at ~uint(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF) {}
 // ----
-// TypeError 6396: (21-94): The base slot of the storage layout must evaluate to a rational number.
+// TypeError 1505: (21-94): The base slot expression cannot be evaluated during compilation.
```

Comment:
I think that users will be confused about why we can't evaluate this, because fundamentally it should be possible. It's documented, but maybe it would not hurt to just say that the constant evaluator is just too limited for this?

```suggestion
// TypeError 1505: (21-94): The base slot expression contains elements that are not yet supported by the internal constant evaluator and therefore cannot be evaluated at compilation time.
```

Reply:
Done.